coordinator/internal/authority/userapi_test.go

```diff
@@ -458,15 +472,227 @@ func TestUserAPIConcurrent(t *testing.T) {
 	wg.Wait()
 }
 
-func newCoordinator() *Authority {
-	return newCoordinatorWithRegistry(prometheus.NewRegistry())
+func TestOutOfBandUpdates(t *testing.T) {
+	require := require.New(t)
+	store := newWatchableStore()
+	hist := history.NewWithStore(slog.Default(), store)
+	a := New(hist, prometheus.NewRegistry(), slog.Default())
+	t.Cleanup(a.Stop)
+
+	// Set an initial manifest.
+	seedShareOwnerKey := testkeys.RSA(t)
+	seedShareOwnerKeyBytes := manifest.MarshalSeedShareOwnerKey(&seedShareOwnerKey.PublicKey)
+
+	mnfst, _, policies := newManifest(t)
+	mnfst.SeedshareOwnerPubKeys = []manifest.HexString{seedShareOwnerKeyBytes}
+	manifestBytes, err := json.Marshal(mnfst)
+	require.NoError(err)
+
+	req := &userapi.SetManifestRequest{
+		Manifest: manifestBytes,
+		Policies: policies,
+	}
+	setManifestResp, err := a.SetManifest(context.Background(), req)
+	require.NoError(err)
+
+	// GetManifest should show that the watcher did not mark the state stale.
+	getManifestResp, err := a.GetManifests(context.Background(), nil)
+	require.NoError(err)
+	require.Len(getManifestResp.Manifests, 1)
+	require.Equal(manifestBytes, getManifestResp.Manifests[0])
+
+	// Manipulate history directly
+	key := a.state.Load().seedEngine.TransactionSigningKey()
+	oldLatest, err := hist.GetLatest(&key.PublicKey)
+	require.NoError(err)
+	transition := &history.Transition{
+		ManifestHash:           sha256.Sum256(manifestBytes),
+		PreviousTransitionHash: oldLatest.TransitionHash,
+	}
+	transitionHash, err := hist.SetTransition(transition)
+	require.NoError(err)
+	nextLatest := &history.LatestTransition{
+		TransitionHash: transitionHash,
+	}
+	require.NoError(hist.SetLatest(oldLatest, nextLatest, key))
+
+	// Wait for the staleness to propagate.
+	require.Eventually(func() bool {
+		return a.state.Load().stale.Load()
+	}, time.Second, 10*time.Millisecond)
+	_, err = a.GetManifests(context.Background(), nil)
+	require.ErrorContains(err, ErrNeedsRecovery.Error())
+
+	// Recovery should succeed.
+	seed, err := manifest.DecryptSeedShare(seedShareOwnerKey, setManifestResp.GetSeedSharesDoc().GetSeedShares()[0])
+	require.NoError(err)
+
+	recoverReq := &userapi.RecoverRequest{
+		Seed: seed,
+		Salt: setManifestResp.GetSeedSharesDoc().GetSalt(),
+	}
+	_, err = a.Recover(rpcContext(seedShareOwnerKey), recoverReq)
+	require.NoError(err)
 }
 
-func newCoordinatorWithRegistry(reg *prometheus.Registry) *Authority {
+func TestStoreRaces(t *testing.T) {
+	require := require.New(t)
+	ctx := context.Background()
+	log := slog.Default()
+
+	store := newWatchableStore()
+	hist := history.NewWithStore(slog.Default(), store)
```

Comment:
```suggestion
	hist := history.NewWithStore(log, store)
```
Or do they have to be different loggers?

Reply:
no, that was just an oversight